lib/cc/engine/rubymotion.rb

```diff
@@ -42,6 +40,29 @@ def run
 
       private
 
+      def files_to_analyze
```

Comment:
It's a good separation of concerns and I can do this. However, I think if we decide to go with a class approach, we should keep it consistent and do the same in every other engine. That would require small changes to engines which already have switched from `exclude_paths` to `include_paths` such as Rubocop, Eslint, and CSSlint.